docs/portal/developer-portal/advanced/Enabling_K3s.md

```diff
@@ -0,0 +1,321 @@
+        }
+       },
+       {
+         "Key": "subgid",
+         "LastModified": "2023-02-21T23:41:43.948000+00:00",
+         "ETag": "\"73032ede132e44d2c1bc567246901737\"",
+         "Size": 145686,
+         "StorageClass": "STANDARD",
+         "Owner": {
+           "DisplayName": "User Service User",
+           "ID": "USER"
+         }
+       }
+     ]
+   }
+
+### Podman Image
+A container image suitable for users should be available in a container image registry accessible from the UAN.
+
+## Configuring with Configuration Framework Service (CFS)
+
+After completing the [Prerequisites](#prerequisites) section, the following are available to proceed with configuring a UAN to run K3s.
+
+- A fully configured UAN
+- An IPAddress start and end range to assign to MetalLB
+- Prepared subuid and subgid files 
+
+### Configuration Files and Playbook
+
+Configuration for K3s and related services are controlled in the following ansible files in the UAN VCS repository:
+```bash
+$ ls uan-config-management/vars/ | egrep "k3s|sshd|helm"
+uan_helm.yml
+uan_k3s.yml
+uan_sshd.yml
+```
+The ansible playbook to install and configure K3s may be found here:
+```bash
+$ ls uan-config-management | grep k3s
+k3s.yml
+```
+
+### Ansible Roles
+
+The following Ansible roles are provided in the `uan-config-management` repository in VCS. There are `README.md` files in each Ansible role directory in this repository that provide further details.
+
+- **uan-k3s-install**: Download and stage K3s assets necessary to initialize and configure k3s on a node
+- **uan-k3s-stage**: Install and configure K3s on a node
+- **uan-helm**: Perform tasks to initialize an environment to install helm charts
+- **uan-haproxy**: Deploy a list of HAProxy charts to the k3s cluster
+- **uan-metallb**: Deploy the MetalLB chart to the k3s cluster
+- **uan-sshd**: Create and enable instances of `sshd` with systemd
+
+### Artifactory Assets
+
+UAN uploads artifacts to deploy to the UAN control-plane node in a new nexus repository:
+
+- uan-2.6.XX-third-party
+
+The following Nexus group repository is created and reference the aforementioned UAN Nexus raw repos.
+
+- uan-2.6-third-party
+
+This repository will contain the installer for K3s, Helm charts for HAProxy and MetalLB, etc.
+
+### Validation Tests 
+
+To validate the K3s cluster once deployed, see the [Validation Checks](#validation-checks) section of this document for details.
+
+## Configuring K3s, MetalLB, HAProxy, and SSHD for use with Podman
+
+Each of the sections below describe how the various comonents deployed to K3s and the UANs may be configured to enable users to SSH to rootless podman containers. As there is no one configuration to fit any one use case, read and understand each section to modify the configuration as needed. Once each section has been completed, see [Deploy K3s to the UAN](#deploy-k3s-to-the-uan).
+
+### MetalLB 
+
+Configure the start and end range for MetalLB `IPAddressPool` in `vars/uan_helm.yml`:
```

Comment:
Can we offer any hard requirements or guidance about the bare minimum number of IPs that need to be in the metallb address pool? Once  the customer knows that, then they'll quickly decide whether it's worth the effort to reclaim some IPs from CSM.

Reply:
Good suggestion.